mods/gen2/scripts.js

```diff
@@ -117,6 +117,134 @@ exports.BattleScripts = {
 		this.singleEvent('AfterMove', move, null, pokemon, target, move);
 		if (!move.selfSwitch && target.hp > 0) this.runEvent('AfterMoveSelf', pokemon, target, move);
 	},
+	tryMoveHit: function (target, pokemon, move, spreadHit) {
```

Comment:
How does this differ from other gens? I'd usually prefer a check for gen in the future-gen script instead of wholesale replacing large functions.

Reply:
Gen II accuracy works more like Gen I than current Gen, where it's in 255 instead of 100. It also fixes the bug from Gen I where it had a 1/256 chance to miss.

Reply:
The "bug" in gen 1 where all moves have a 1/256 chance to miss, is from the games themselves, not PS.

Reply:
Yes that I know, but Gen II did not have it so I fixed it when I transferred the code from Gen I to Gen II.

Reply:
Ah, I misread your comment then sorry.